editor/src/core/model/element-template-utils.spec.tsx

```diff
@@ -98,13 +98,13 @@ describe('guaranteeUniqueUids', () => {
     ]
     const fixedElements = guaranteeUniqueUids(exampleElements, emptySet())
 
-    const child0PropsOptic = fromArrayIndex<JSXElementChild>(0)
+    const child0PropsOptic = fromArrayIndex<JSXElementChild | null>(0)
```

Comment:
With this then we don't need to change `isJSXElement`, which can also cause issues when composing it when the value isn't null.
```suggestion
    const child0PropsOptic = fromArrayIndex<JSXElementChild>(0).compose(notNull())
```